packages/api/source/models/ActivityLogModel.ts

```diff
@@ -19,6 +19,10 @@ const activityLogSchema = new Schema(
         context: {
             type: Object,
         },
+        createdAt: {
+            type: Date,
+            immutable: true,
+        },
```

Comment:
Why doesn't this have a default value?

Reply:
It will be automatically added as {timestamps:true}